clc/modules/object-storage-common/src/main/java/com/eucalyptus/objectstorage/msgs/ObjectStorage.groovy

```diff
@@ -81,8 +94,12 @@ public class ObjectStorageStreamingResponseType extends StreamedBaseMessage {
 public class ObjectStorageRequestType extends BaseMessage {
   protected Date timeStamp;
   BucketLogData logData;
+  // This is sometimes the bucket name, sometimes the bucket UUID.
+  //TODO: Stop doing that! Make separate fields.
```

Comment:
Not sure I understand the comment here. Bucket UUID is an internal construct to OSG and backend. OSG generates uuids for user input bucket (and object) names before implementing them on the backend